src/components/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Why do you need to disable it?

Reply:
Well there is a reason for it. You are using <div> as a button 🙂 so maybe it is worth to thing does this element shouldn't be an button, styled as you need 😁 ... But overally I would ignore that rule in .eslintrc if you don't wanna to change it 🙂